app/presenters/education_navigation_ab_testable.rb

```diff
@@ -0,0 +1,17 @@
+module EducationNavigationABTestable
```

Comment:
@mobaig as discussed, I've moved the module into the `models` folder, but will probably leave the rest as-is, given that the code will be removed again after the A/B test is over.